std/http/file_server.ts

```diff
@@ -22,22 +23,42 @@ interface EntryInfo {
   name: string;
 }
 
+interface FileServerArgs {
+  _: string[];
+  // -p --port
+  p: number;
+  port: number;
+  // --cors
+  cors: string;
+  // -h --help
+  h: string;
+  help: string;
+}
+
 const encoder = new TextEncoder();
-const serverArgs = args.slice();
-let CORSEnabled = false;
-// TODO: switch to flags if we later want to add more options
-for (let i = 0; i < serverArgs.length; i++) {
-  if (serverArgs[i] === "--cors") {
-    CORSEnabled = true;
-    serverArgs.splice(i, 1);
-    break;
-  }
+
+const serverArgs = parse(args) as FileServerArgs;
+
+const CORSEnabled = serverArgs.cors ? true : false;
+const target = posix.resolve(serverArgs._[1] || "");
```

Comment:
`serverArgs._[1]` shouldn't this be `serverArgs._[0]`?

Reply:
"0" is itself, "1" is the first value typed

Reply:
Blah, obviously.